augur/__init__.py

```diff
@@ -76,6 +78,15 @@ def run(argv):
     except RecursionError:
         print("FATAL: Maximum recursion depth reached. You can set the env variable AUGUR_RECURSION_LIMIT to adjust this (current limit: {})".format(sys.getrecursionlimit()))
         sys.exit(2)
+    except Exception:
```

Comment:
> @tsibley : The best way to skip both `SystemExit` and `KeyboardInterrupt` btw is to catch `Exception` instead of everything. See the [exception hierarchy](https://docs.python.org/3/library/exceptions.html#exception-hierarchy).

Oh cool! Done.